src/v2/Components/AddToCalendar/helpers.ts

```diff
@@ -0,0 +1,103 @@
+import { DateTime } from "luxon"
+import { stripTags } from "underscore.string"
+import { CalendarEventProps } from "v2/Components/AddToCalendar/AddToCalendar"
+
+/** @returns URL to create a new Google Calendar event */
+export const generateGoogleCalendarUrl = ({
+  title,
+  startDate,
+  endDate,
+  description = "",
+  address = "",
+  href,
+  liveAuctionUrl,
+}: CalendarEventProps): string => {
+  const start = formatCalendarDate(startDate)
+  // If there is no endDate, set to 1 hour after start
```

Comment:
I wonder if we should make this shorter, e.g. 15m, so that people don't accidentally think the sale is going live up to an hour after it's already started.

Reply:
@eessex If we're going to keep the title the same then this seems fine, we can also add it as a future improvement.

Reply:
For live sales the timing is passed in as `endDate`, which is set to 4 hours (this was the existing implementation for live sales).